scripts/pelorus-operator-patches/02_makefile_image_url.diff

```diff
@@ -1,7 +1,7 @@
 --- Makefile.original	2023-04-06 10:52:02.085396187 +0200
 +++ Makefile	2023-04-06 11:03:30.396214039 +0200
-@@ -47,7 +47,7 @@
- endif
+@@ -51,7 +51,7 @@
+ OPERATOR_SDK_VERSION ?= v1.31.0
```

Comment:
this works if newest Operator SDK version, 1.31.0, but because of the change, it would need to be updated every time we update its version

My idea is delete this folder, but want opinions